pkg/gardenlet/operation/botanist/secrets_test.go

```diff
@@ -293,6 +293,8 @@ var _ = Describe("Secrets", func() {
 
 			Context("observability credentials", func() {
 				It("should generate the password and sync it to the garden", func() {
+					botanist.Shoot.WantsAlertmanager = true
```

Comment:
Nit: There is one more occurrence in an e2e can be adapted to use the new annotation: https://github.com/gardener/gardener/blob/10d75787a132bf39c408b74372f5d8c045fa8f4b/test/e2e/gardener/shoot/create_rotate_delete.go#L58-L60